src/Board.coffee

```diff
@@ -83,6 +85,18 @@ class Board
   
   revealAll: ->
     tile.reveal() for tile in @tiles
+    
+  revealPitTiles: ->
+    tile.reveal() for tile in @tiles when tile.kind is 'pit'
+    console.log('revealPitTiles() is called here!')
+    message =
+      event: 'draw'
+    @game.sendMessage message, @, @game
```

Comment:
you probably don't need to issue a draw message here. would be better to have the game redraw after calls this method